index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fitno">
+    <title>D4niel J Portofolio</title>    
+    <link rel="stylesheet" type="text/css" href="css/normalize.css" media="screen"/>
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,200;0,300;0,400;0,500;0,600;0,700;0,800;1,200;1,300;1,400;1,500;1,600;1,700;1,800&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="css/styles.css" media="screen"/>
+    <script src="https://kit.fontawesome.com/d5e3631d2a.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <nav>
+      <div id="nav-container">
+        <div id="logo">
+          D4NIEL J
+        </div>
+        <div id="menu">
+          <div class="menu-line"></div>
+          <div class="menu-line"></div>
+          <div class="menu-line"></div>
+        </div>
+      </div>
+    </nav>
+    <header>
+      <h1 id="header-title">I’m Daniel<br>
+         Glad to see you!</h1>
+      <p id="header-paragraph">
+        I’m a software developer! 
+        I can help you build a product, 
+        feature or website Look through 
+        some of my work and experience! 
+        If you like what you see and have 
+        a project you need coded, 
+        don’t hesitate to contact me.
+      </p>
+      <div id="social-media">
+        <h2>LET’S CONNECT</h2>
+        <div id="social-media-container">
+          <i class="fab fa-twitter"></i>
+          <i class="fab fa-github"></i>
+          <i class="fab fa-instagram"></i>
```

Comment:
- [ ] The list of icons should use a `ul` tag to group them, and `li` per icon; each icon has a link (a tag).